stock_removal_location_by_priority/views/stock_location_view.xml

```diff
@@ -10,7 +10,7 @@
         <field name="inherit_id" ref="stock.view_location_form"/>
         <field name="arch" type="xml">
             <field name="active" position="after">
-                <field name="removal_priority"/>
+                <field name="removal_priority" group="removal_priority_active"/>
```

Comment:
@gustavovalverde if you are in a hurry and you don't use multi-company, by removing in this view `group="removal_priority_active"` and checking the box *Use 'Removal Priority' in Locations* in the setting it will work ok for you.

Reply:
I'm indeed moving to multicompany , what could I expect in a multicompany environment?